assets/src/edit-story/app/font/defaultFonts.js

```diff
@@ -16,7 +16,7 @@
 
 // @todo: only provide font family, and use font provider to fill the rest.
 export const TEXT_ELEMENT_DEFAULT_FONT = {
-  family: 'Roboto',
+  family: 'Google Sans',
```

Comment:
I don't think we want to change the default font used in stories themselves...

Reply:
1. Google Sans is a special font for use in Google products (e.g. in UI), not intended for anyone to use in their web content.
2. It's not an option in the font picker.